firmware/boron-ins-fsm/src/stateMachine.h

```diff
@@ -60,6 +63,9 @@ typedef void (*StateHandler)();
 // declaring the state handler pointer as extern so .ino file can use it
 extern StateHandler stateHandler;
 
+// time in ms since start of device of last heartbeat publish
+extern unsigned long lastHeartbeatPublish;
```

Comment:
I think this variable is only ever referenced inside `stateMachine.cpp` so it doesn't need to be in the `.h`. Or did it get added somewhere?